README.md

```diff
@@ -84,7 +84,7 @@ The following table shows which Minters and MinterFilters are compatible with wh
 | V0                       | -              | (legacy minter)                                                                                                 | -                                                                           |
 | V1, V1_PRTNR             | MinterFilterV0 | MinterSetPriceV1<br>MinterSetPriceERC20V1<br>MinterDAExpV1<br>MinterDALinV1<br>MinterMerkleV0<br>MinterHolderV0 | MinterSetPriceV0<br>MinterSetPriceERC20V0<br>MinterDAExpV0<br>MinterDALinV0 |
 | V2 (PBAB)                | -              | (various PBAB minters)                                                                                          | -                                                                           |
-| V3                       | MinterFilterV1 | MinterSetPriceV2<br>MinterSetPriceERC20V2<br>MinterDAExpV2<br>MinterDALinV2<br>MinterMerkleV1<br>MinterHolderV1 | -                                                                           |
+| V3                       | MinterFilterV1 | MinterSetPriceV2<br>MinterSetPriceERC20V2<br>MinterDAExpV2<br>MinterDALinV2<br>MinterMerkleV2<br>MinterHolderV1 | -                                                                           |
```

Comment:
This diff seems off to me–could you clarify please?

Reply:
Oh sorry, I was assuming since we haven't deployed core V3 to prod then our V1 minter should be replaced by V2, instead of listing both in the ReadMe. Is it better if I list both V1 and V2 instead of replacing this one?